ast/src/analyzed/mod.rs

```diff
@@ -782,6 +782,53 @@ impl<T> LookupIdentity<T> {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct PhantomLookupIdentity<T> {
```

Comment:
please add a docstring explaining that it is only used for witness generation.

Reply:
done